src/index.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
-const _ = require('lodash');
 const core = require('./core');
 const RippleAPI = require('./api');
 
-module.exports = _.assign({}, core, {RippleAPI: RippleAPI});
+module.exports = {
+  RippleAPI,
+  _DEPRECATED: core   // WARNING: this will be removed soon
+};
```

Comment:
Oh right, it's because we have a `test/node_modules/ripple-lib` symbolic link to core: https://github.com/clark800/ripple-lib/blob/deprecate/test/node_modules/ripple-lib